src/System.Windows.Extensions/tests/System/Drawing/ImageFormatConverterTests.cs

```diff
@@ -0,0 +1,229 @@
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TestConvertFrom_ImageFormatToString()
+        {
+            Assert.Equal(_imageFmt, (ImageFormat)_imgFmtConv.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp.ToString()));
+            Assert.Equal(_imageFmt, (ImageFormat)_imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp.ToString()));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void TestConvertFrom_ThrowsNotSupportedException()
+        {
+            Assert.Throws<NotSupportedException>(() => _imgFmtConv.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConv.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp.Guid));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConv.ConvertFrom(null, CultureInfo.InvariantCulture, new object()));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConv.ConvertFrom(null, CultureInfo.InvariantCulture, 10));
+                          
+            Assert.Throws<NotSupportedException>(() => _imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, ImageFormat.Bmp.Guid));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, new object()));
+            Assert.Throws<NotSupportedException>(() => _imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, 10));
+        }
+
+        private ImageFormat ConvertFromName(string imgFormatName)
+        {
+            return (ImageFormat)_imgFmtConvFrmTD.ConvertFrom(null, CultureInfo.InvariantCulture, imgFormatName);
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void ConvertFrom_ShortName()
+        {
+            Assert.Equal(ImageFormat.Bmp, ConvertFromName("Bmp"));
+            Assert.Equal(ImageFormat.Emf, ConvertFromName("Emf"));
+            Assert.Equal(ImageFormat.Exif, ConvertFromName("Exif"));
+            Assert.Equal(ImageFormat.Gif, ConvertFromName("Gif"));
+            Assert.Equal(ImageFormat.Tiff, ConvertFromName("Tiff"));
+            Assert.Equal(ImageFormat.Png, ConvertFromName("Png"));
+            Assert.Equal(ImageFormat.MemoryBmp, ConvertFromName("MemoryBmp"));
+            Assert.Equal(ImageFormat.Icon, ConvertFromName("Icon"));
+            Assert.Equal(ImageFormat.Jpeg, ConvertFromName("Jpeg"));
+            Assert.Equal(ImageFormat.Wmf, ConvertFromName("Wmf"));
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Suppor to convert image format from long name was added to .NET Core directly.")]
```

Comment:
Nit: Suppor => support
```suggestion
        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Support to convert image format from long name was added to .NET Core directly.")]
```